src/components/dashboard-shortcut/dashboard-shortcut.scss

```diff
@@ -0,0 +1,80 @@
+/**
+* @prop --dashboard-shortcut-border-radius: defines the radius of corners of the shortcut. Defaults to `1rem`
+* @prop --dashboard-shortcut-icon-color: defines the fill color of the shortcut icon. Defaults to `--contrast-1000`
+* @prop --dashboard-shortcut-label-color: defines the color of the shortcut label. Defaults to `--contrast-11000`
+* @prop --dashboard-shortcut-background-color: defines the backgrounds color of the shortcut icon. Defaults to `--contrast-100`
+* @prop --dashboard-shortcut-badge-text-color: Text color of the notification badge. Defaults to `--color-white`
+* @prop --dashboard-shortcut-badge-background-color: Background color of the notification badge. Defaults to `--color-red-default`
```

Comment:
Is this overkill?
I mean, it would already be still possible to style the badge, using `limel-badge`'s own CSS properties. But the consumer should be "one of us" to know that they are available. Or have a deep understanding of how css variables can penetrate the shadow doms. 

On the other hand, I have never seen the notification badges, in any color in other than white text on red background. Do we want people to change their colors?

Maybe we want, if the color is trying to attract attention for some reason. I can't really decide.

Reply:
> Do we want people to change their colors?

I can see arguments for either option, but I prefer not restricting possibilities unless necessary, and I prefer not adding code unless necessary. When considering adding a restriction that would require extra code, there better be a _very_ good reason, imho 😝

I was about to suggest that rather than adding this extra layer of CSS variables, we could just document the variables available from `limel-badge` here too, but then I realised that's exactly the kind of "leaking" if information about the internal implementation we want to avoid, as it becomes a problem if and when we want to change the internal implementation. So I say we keep these CSS variables! 👍